python/openmldb_tool/diagnostic_tool/diagnose.py

```diff
@@ -108,6 +114,10 @@ def insepct_online(args):
     assert not fails, f"unhealthy tables: {fails}"
     print(f"all tables are healthy")
 
+    if flags.FLAGS.db:
```

Comment:
It's not good to understand, why `--db` triggers distribution show?
You chould use another option, e.g. `tool inspect --dist --db 'db1'`